code/controllers/ContentController.php

```diff
@@ -87,42 +88,58 @@ public function Page($link) {
 
 	public function init() {
 		parent::init();
-		
+
+		$recordExists = ($this->dataRecord && $this->dataRecord->exists());
```

Comment:
Honestly I can't see where it could, after looking through this all again. I think it's fine, but check my note below on extensions on the dataRecord. :D